python/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# ******************************************************************************
+# Copyright 2017-2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+cmake_minimum_required (VERSION 3.1)
+
+project (pyngraph)
+
+include(ExternalProject)
+
+ExternalProject_Add(
+        pybind11
+        GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
```

Comment:
Don't you need to use a specific version? For example a release tag or a git commit hash?

Reply:
Yes, it needs to be specific about which pybind11 version/tag to use